charts/sda-svc/test/trigger-ingest.py

```diff
@@ -63,11 +68,12 @@
 
     # If client verification is required
     if certfile.exists():
-        assert( keyfile.exists() )
+        assert(keyfile.exists())
```

Comment:
not ideal maybe we can check `if certfile.exists() and keyfile.exists():` if not we raise execption or do an sys.exit() and an error code from: https://docs.python.org/3.8/library/os.html#os._exit

Reply:
fair enough, then we can have that as part of another PR